cmd/doctor.go

```diff
@@ -510,16 +510,20 @@ func runDoctorCheckDBConsistency(ctx *cli.Context) ([]string, error) {
 	if err := models.NewEngine(context.Background(), migrations.EnsureUpToDate); err != nil {
 		upToDate = false
 		results = append(results, err.Error())
+		if ctx.Bool("fix") {
+			results = append(results, "DB Version does not match the current one, cant fix Consistency")
```

Comment:
```suggestion
			results = append(results, "Warning: model version on the database does not match the current Gitea version. Model consistency can be checked but not fixed until the database is upgraded.")
```
😄 
Maybe we can run the _checks_ anyway. This is to cover for cases where users never run the doctor without the `--fix` flag.

Reply:
I mean we should not just return after this message. We should perform the checks anyway, just not the fixing.